doc/classes/PhysicsServer3D.xml

```diff
@@ -1609,22 +1614,25 @@
 		<constant name="SHAPE_CAPSULE" value="4" enum="ShapeType">
 			The [Shape3D] is a [CapsuleShape3D].
 		</constant>
-		<constant name="SHAPE_CYLINDER" value="5" enum="ShapeType">
+		<constant name="SHAPE_CONE" value="5" enum="ShapeType">
```

Comment:
Adding an enum value in the middle like this breaks compatibility, I can say this for certain right off the bat.

Reply:
What I mean is, this enum constant should be added at the very end of the list. It's fine if it's out of order, compatibility is more important.

Reply:
It seems that the checks want the documentation to be in alphabetical? The check is failing and prefers it in position 5 rather than at the end of the list.

Reply:
It's not enough to change only the documentation, you also need to change `servers/physics_server_3d.h`.